### PR TITLE
Fix broken links in blogs

### PR DESCRIPTION
**Pull request by matthewshaver**
## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/blog/2019-05-01-how-we-structure-dbt-projects.md [hunk 1]
```diff
@@ -37,8 +37,6 @@ In comparison, the (recently updated) [best practices](/guides/best-practices) r
 *   Consider the information architecture of your data warehouse
 *   Separate source-centric and business-centric transformations
 
-If you want to see what the code for one of our projects looks like, check out [this demonstration dbt project](https://github.com/dbt-labs/dbt-learn-demo/tree/day2-dbt-training/models).
-
```
Comment (nghi-ly) [inline website/blog/2019-05-01-how-we-structure-dbt-projects.md:41]: flagging just in case, is this removal intentional (versus a replacement)?

File: website/blog/2021-09-11-sql-dateadd.md [hunk 2]
```diff
@@ -142,6 +142,3 @@ And it’s actually quite a simple 31-line macro ([source here](https://github.c
 Enjoy! FYI I've used dateadd macro in dbt-utils on BigQuery, Postgres, Redshift and Snowflake, but it likely works across most other warehouses.
 
 *Note: While `dbt_utils` doesn't support Databricks by default, you can use other packages that [implement overrides](/reference/dbt-jinja-functions/dispatch#overriding-package-macros) as a workaround.*
-
-*This [spark_utils package](https://github.com/dbt-labs/spark-utils/blob/main/macros/dbt_utils/cross_db_utils/dateadd.sql) can help you implement the override needed to add support for Databricks dateadd*
-
```
Comment (nghi-ly) [inline website/blog/2021-09-11-sql-dateadd.md:147]: just in case, flagging this removal too to see if it was deliberate.

File: website/blog/2021-11-29-dbt-airflow-spiritual-alignment.md [hunk 3]
```diff
@@ -232,7 +232,7 @@ To set up Airflow and dbt Cloud, you can:
 
 ![airflow dbt run select](/img/blog/2021-11-29-dbt-airflow-spiritual-alignment/airflow-connection-ID.png)
 
-3. Set up your Airflow DAG similar to [this example](https://github.com/apache/airflow/blob/main/airflow/providers/dbt/cloud/example_dags/example_dbt_cloud.py).
+3. ~Set up your Airflow DAG similar to this example.~
```
Comment (nghi-ly) [inline website/blog/2021-11-29-dbt-airflow-spiritual-alignment.md]:
```suggestion
3. Set up your Airflow DAG similar to this example.
```

Comment (nghi-ly) [reply website/blog/2021-11-29-dbt-airflow-spiritual-alignment.md]: question: which example is step 3 referring to? is it the PNG graph referenced on line 239 below?

File: website/blog/2021-11-29-the-missing-role-of-design-in-analytics.md [hunk 4]
```diff
@@ -35,7 +35,7 @@ Painting with broad strokes, design is generally not something that is a top pri
 
 These more or less boil down to data teams running like service teams rather than [product teams](https://locallyoptimistic.com/post/run-your-data-team-like-a-product-team/)—when you always give the squeakiest wheel the grease, it’s impossible to put in the strategic effort that design thinking requires.
 
-One solution I proposed back in 2019 is hiring a [data product manager](https://www.linkedin.com/pulse/why-your-organization-may-need-data-product-manager-seth-rosen/), which seems to be picking up a bit of steam. But what would that person actually do?
+One solution I proposed back in 2019 is hiring a [data product manager](https://www.hashpath.com/2019/11/why-your-organization-may-need-a-data-product-manager/), which seems to be picking up a bit of steam. But what would that person actually do?
```
Comment (nghi-ly) [inline website/blog/2021-11-29-the-missing-role-of-design-in-analytics.md:38]: nice!

**Review by nghi-ly — COMMENTED** [review]
@matthewshaver : thanks so much for getting this wrapped up! looks great. left a few clarifying questions.

File: website/blog/2021-09-11-sql-dateadd.md (hunk 2 above)
Comment (matthewshaver) [reply website/blog/2021-09-11-sql-dateadd.md:147]: This was deliberate as the page is gone and I can't find a suitable replacement in their Git directory.  Seems it no longer exists.

File: website/blog/2021-11-29-dbt-airflow-spiritual-alignment.md (hunk 3 above)
Comment (matthewshaver) [reply website/blog/2021-11-29-dbt-airflow-spiritual-alignment.md]: @nghi-ly This used to be a hyperlink - I removed the link and put strikethrough text to note that it's no longer there.  I'm not sure why this doesn't reflect the link removal

Comment (nghi-ly) [reply website/blog/2021-11-29-dbt-airflow-spiritual-alignment.md]: it doesn't render the strikethrough, which is odd: 

https://deploy-preview-2315--docs-getdbt-com.netlify.app/blog/dbt-airflow-spiritual-alignment

Screen Shot 2022-10-26 at 1 38 55 PM

File: website/blog/2021-09-11-sql-dateadd.md (hunk 2 above)
Comment (mirnawong1) [reply website/blog/2021-09-11-sql-dateadd.md:147]: @matthewshaver is it this link: https://github.com/dbt-labs/dbt-utils/blob/eb1ec81c7bc58c3fbcd2cbcf4216b276ccd893ba/macros/cross_db_utils/dateadd.sql?

there's also another dateadd.sql file here: https://github.com/dbt-labs/spark-utils/blob/0.3.0/macros/dbt_utils/cross_db_utils/dateadd.sql but i'm not sure which is more up to date

Comment (matthewshaver) [reply website/blog/2021-09-11-sql-dateadd.md:147]: @mirnawong1 - You found it!  I swear I looked everywhere and all I found was that first one.  But the second should cover it.
Thank you!

File: website/blog/2021-11-29-dbt-airflow-spiritual-alignment.md (hunk 3 above)
Comment (matthewshaver) [inline website/blog/2021-11-29-dbt-airflow-spiritual-alignment.md]:
```suggestion
3. ~~Set up your Airflow DAG similar to this example.~~
```

File: website/blog/2021-10-15-october-21-product-update-email.md [hunk 5]
```diff
@@ -78,10 +78,10 @@ The Model Timing tab in dbt Cloud highlights models taking particularly long to
...
 -   [Modeling behavioral data with Snowplow and dbt](https://get.snowplowanalytics.com/wbn/dbt-and-snowplow/data-modeling/?utm_campaign=Monthly%20Product%20Updates&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz-_wfy8vfjMjwQ7o8TXEOVz-oXI35iVcVP1HtAvriVHfJoAd1IcsP-MCww6vJUDlvAfiuQjZ) (coming up on 10/27).
     Our own Sanjana Sen joins the Snowplow team to talk modeling Snowplow event data in dbt -- including how to structure your data models, best practices to follow, and key pitfalls to avoid.
--   [How Blend Eliminated Data Silos with dbt and Hightouch](https://hightouch.io/dbt-hightouch-blend-event/?utm_campaign=Monthly%20Product%20Updates&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz-_wfy8vfjMjwQ7o8TXEOVz-oXI35iVcVP1HtAvriVHfJoAd1IcsP-MCww6vJUDlvAfiuQjZ) (coming up 10/28).
+-   ~How Blend Eliminated Data Silos with dbt and Hightouch (coming up 10/28).~
```
Comment (nghi-ly) [inline website/blog/2021-10-15-october-21-product-update-email.md]:
```suggestion
-   How Blend Eliminated Data Silos with dbt and Hightouch (coming up 10/28).
```

**Review by nghi-ly — COMMENTED** [review]
fantastic! just one pair of straggling tildes

File: website/blog/2021-10-15-october-21-product-update-email.md (hunk 5 above)
Comment (matthewshaver) [inline website/blog/2021-10-15-october-21-product-update-email.md]:
```suggestion
- How Blend Eliminated Data Silos with dbt and Hightouch.

**Review by nghi-ly — APPROVED** [review]
:shipit:
